web/app/.eslintrc

```diff
@@ -84,6 +84,6 @@
 
     // These rules represent best practices that we're not adhering to, and should be enabled
     "jsx-a11y/click-events-have-key-events": 0,
-    "no-param-reassign": 0
+    "no-param-reassign": 1
```

Comment:
I think it's better to just delete this line to enable that rule

Reply:
@alpeb  I will do the same, thank you for the suggestion :)

Reply:
By using this line `"no-param-reassign": ["error", { "props": false }] ` , we can reassign the properties of the objects that are passed as parameters.

Reply:
That looks good :+1: